puppet/modules/web/files/stagingrpm/HEADER.html

```diff
@@ -0,0 +1,16 @@
+<h1>stagingrpm.theforeman.org</h1>
```

Comment:
Do we want to stick with this paradigm of `stagingrpm` or something like `staging.rpm` or `rpm.staging`?

Reply:
If we change it, then I'd use the `staging` subdomain so you'd have both `rpm.staging` and `deb.staging`.

Reply:
@evgeni thoughts?

Reply:
I like the staging subdomain idea!

Probably should also apply this to archivedeb then, but, alas, different incarnation.

Reply:
Ah, of course, it's never as easy as it seems.
The certificate we have on Fastly is only valid for `*.theforeman.org` and won't work for `*.staging.theforeman.org`.
Nothing that we can't solve, but probably should be better planned in a dedicated change?

Reply:
Before or after this and your change?

Reply:
I'd do that after, it's cosmetic and we have more pressing issues right now